config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
I can try to add this.
BTW, I have raised https://github.com/clearcontainers/tests/issues/198 because semaphore is failing because q35 is not installed. I was thinking about installing the q35 version with a different name since we will get current tests failing otherwise (our version of qemu does not support q35 and the q35 version does not support pc-lite). And from this PR, I would specify this different name in the Makefile. What do you think ?